### ISSUE TITLE
How to cite?

### ISSUE THREAD

**Issue by Querela** [internetarchive/heritrix3#463]
Hi,

I'm writing a paper and would like to cite heritrix. What/Who/... do I need to reference? I found the following:
- [Mohr, G., Stack, M., Rnitovic, I., Avery, D., & Kimpton, M. (2004, July). An Introduction to Heritrix. In 4th International Web Archiving Workshop (pp. 109-115).](http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.676.6877&rep=rep1&type=pdf) ?

Suggestion: add a section with bibtex code to either the readme or the wiki for future users (maybe [here](https://github.com/internetarchive/heritrix3/wiki/Users%20of%20Heritrix)).

**Comment by ato** [comment]
Feel free to send a pull request updating the README or documentation as you feel would be appropriate.

I'm no academic and don't know what the normal practice is but citing that paper seems reasonable to me and looking on Google Scholar it does seem to be what most people reference.

**Comment by cgr71ii** [comment]
I agree that I'd be useful to have a reference somewhere in the github page. The paper you mentioned is a little bit old (I haven't read it, but I'm sure that doesn't reflect the current status of Heritrix), but it seems to be the official, and 2021 and 2022 papers cited it. Have you found another reference not that old?

Anyway, here is the bibtext (I'm not sure if you asked for it or just asked for other reference as I've pointed out):

```
@inproceedings{mohr2004introduction,
  title={Introduction to heritrix},
  author={Mohr, Gordon and Stack, Michael and Avery, Dan and Kimpton, Michele},
  organization={Citeseer}
}
```
